specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/ServerTrustCertificates.json

```diff
@@ -0,0 +1,297 @@
+      "get": {
+        "tags": [
+          "ServerTrustCertificates"
+        ],
+        "description": "Gets a list of server trust certificates that were uploaded from box to the given Sql Managed Instance.",
+        "operationId": "ServerTrustCertificates_ListByInstance",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a list of server trust certificates.",
+            "schema": {
+              "$ref": "#/definitions/ServerTrustCertificatesListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Gets a list of server trust certificates on a given server.": {
+            "$ref": "./examples/ServerTrustCertificatesListByInstance.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServerTrustCertificateProperties": {
+      "description": "The properties of a server trust certificate.",
+      "type": "object",
+      "properties": {
+        "publicBlob": {
+          "description": "The certificate public blob",
+          "type": "string"
+        },
+        "thumbprint": {
+          "description": "The certificate thumbprint",
+          "type": "string",
+          "readOnly": true
+        },
+        "certificateName": {
+          "description": "The certificate name",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ServerTrustCertificate": {
+      "description": "Server trust certificate imported from box to enable connection between box and Sql Managed Instance.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServerTrustCertificateProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ServerTrustCertificatesListResult": {
+      "description": "A list of server trust certificates in instance.",
```

Comment:
What does 'server trust certificate' mean?

Reply:
I would change this to:
"A list of the server trust certificates which are used for secure communication between SQL On-Prem instance and the given Sql Managed Instance"

Reply:
I will directly update the description in the swagger PR to pass the review.
@nemijato will send out a PR to fix it in SQL repo as upstream?

Reply:
done